package.json

```diff
@@ -15,17 +15,17 @@
   },
   "repository": {
     "type": "git",
-    "url": "git+https://github.com/singnet/alpha-blockchain.git"
+    "url": "git+https://github.com/singnet/platform-contracts.git"
   },
   "author": "SingularityNET",
   "license": "MIT",
   "bugs": {
-    "url": "https://github.com/singnet/alpha-blockchain/issues"
+    "url": "https://github.com/singnet/platform-contracts/issues"
   },
-  "homepage": "https://github.com/singnet/alpha-blockchain#readme",
+  "homepage": "https://github.com/singnet/platform-contracts#readme",
   "dependencies": {
     "openzeppelin-solidity": "^1.11.0",
-    "singularitynet-token-contracts": "1.0.0",
+    "singularitynet-token-contracts": "2.0.0",
```

Comment:
2.0.0 or 1.1.0? is it breaking because the packaging is breaking? think you mentioned this already

Reply:
So I was torn but the new packaging will break consumers despite the actual code not changing, so I upped the major version to represent that it's not backwards compatible